src/components/onboarding/onboarding.css

```diff
@@ -69,14 +69,14 @@
 
 :global .joyride-hole {
   border-radius: 4px;
-  -webkit-box-shadow: 0 0 0 9999px rgba(0,26,62, 0.5), 0 0 15px rgba(0,26,62, 0.5);
-  box-shadow: 0 0 0 9999px rgba(0,26,62, 0.5), 0 0 15px rgba(0,26,62, 0.5);
+  -webkit-box-shadow: 0 0 0 9999px rgba(0,26,62, 0.9), 0 0 15px rgba(0,26,62, 0.9);
```

Comment:
We don't need browser prefix like '--webkit'. please remove them. read [item number 5](https://github.com/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md) in our css guide for more info.